ci_framework/roles/dlrn_report/tasks/create_hash_info_file.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Write out the hash_info file
+  ansible.builtin.copy:
+    dest: "{{ cifmw_dlrn_report_workspace }}/{{ cifmw_dlrn_report_hash_info_file_name }}"
+    content: |
+      # Do these vars need to start with cifmw_dlrn_report_?
+      export DLRNAPI_URL="{{ dlrnapi_url }}"
+      export RELEASE="{{ osp_release | default(openstack_release | default(release)) }}"
+      export FULL_HASH="{{ full_hash }}"
+      export COMMIT_HASH="{{ commit_hash | default('') }}"
+      export DISTRO_HASH="{{ distro_hash | default('') }}"
+      export COMPONENT_NAME="{{ component | default('') }}"
```

Comment:
We can use repo-setup-get-hash tool to get these info and in ci-framework repo-setup role, we can add a task to generate a dict with all the above info.

Here is the example of repo-setup-get-hash tool to get all the above info:
```
repo-setup on  fix [?] via 🐍 v3.11.4 (.venv) 
❯ repo-setup-get-hash --component baremetal --os-version rhel9 --tag current --release osp18 --dlrn-url https://xx.redhat.com --json | jq
{
  "commit_hash": "7bb46729bdb30e3dce9e03e0a7db9952ec270874",
  "distro_hash": "1e2edf6f05b4aeb5669b22c02e5709f15152b1b4",
  "full_hash": "7bb46729bdb30e3dce9e03e0a7db9952ec270874_1e2edf6f",
  "extended_hash": "86db92a500ac23ee4aa8c9df79a16f9150709289_7e84bfe659458425f7580ecb62c7f8ad6ff90823",
  "dlrn_url": "https://xx.redhat.com/rhel9-osp18/component/baremetal/current/commit.yaml",
  "os_version": "rhel9",
  "release": "osp18",
  "component": "baremetal",
  "tag": "current"
}

repo-setup on  fix [?] via 🐍 v3.11.4 (.venv) 
❯ repo-setup-get-hash --os-version rhel9 --tag current --release osp18 --dlrn-url https://xx.redhat.com --json | jq
{
  "commit_hash": null,
  "distro_hash": null,
  "full_hash": "94e9c3a59211652babbdcebfba4d56d2",
  "extended_hash": null,
  "dlrn_url": "https://xx.redhat.com/rhel9-osp18/current/delorean.repo.md5",
  "os_version": "rhel9",
  "release": "osp18",
  "component": null,
  "tag": "current"
}
```